misc/config_tools/configurator/packages/configurator/src/pages/Config/ConfigForm/CustomWidget/IVSHMEM_REGION.vue

```diff
@@ -71,7 +73,7 @@
                   <template #trigger>
                       <IconInfo/>
                   </template>
-                  <span v-html="this.VMConfigType.properties.name.description"></span>
+                  <span v-html="this.IVSHMEMVM.properties.VM_NAME.description"></span>
```

Comment:
We seem to be using an underscore (`_`) throughout the rest of this file to separate `IVSHMEM` and `VM`. Any reason we shouldn't do it here too?
```suggestion
                  <span v-html="this.IVSHMEM_VM.properties.VM_NAME.description"></span>
```

Reply:
It's a name from scenario.json, I use it directly:
		"IVSHMEMVM": {
			"type": "object",
			"required": [
				"VM_NAME",
				"VBDF"
			],

Reply:
Why not use the same format than the others, i.e. `IVSHMEM_VM`?

Reply:
Ok, will change to IVSHMEM_VM.